guides/working-with-examples.md

```diff
@@ -0,0 +1,47 @@
+# Working with Examples
+
+Adding meaningful examples can greatly enrich the documentation of components in the Cashmere library. Here are some things to remember when creating or modifying examples.
+
+## Examples files
+
+The following files are used in the creation of examples for the Cashmere documentation website.
+
+### Files you may need to modify
+
+-   `projects/cashmere-examples/src/lib/{EXAMPLE_NAME}` - the folder containing all files for an example
+-   `src/app/core/document-items.service.ts` - the file that adds the example to the appropriate component page in the docs site
+
+### Files used in the build process (which you shouldn't need to modify)
+
+-   `src/assets/docs/examples` - the output directory for StackBlitz example project files
+-   `projects/cashmere-examples/src/lib/cashmere.module.ts` - if your example contains a module file, you will need to import this CashmereModule
```

Comment:
This probably belongs in the "Files you may need to modify"

Reply:
Actually, no.  All this file does is import the generated Cashmere module, which is copied from `src/app/shared/cashmere.module.ts` to make it so there is only one CashmereModule file needing to be touched.
I probably should, however add `src/app/shared/cashmere.module.ts` to the list of files you may need to modify.

Reply:
Fixed in latest push